doc/md/_markdown-tests/examples/02-helm-values.txt

```diff
@@ -0,0 +1,357 @@
+# Set $HOME because:
+#   - Holos uses it for tempfiles during 'holos render platform'
```

Comment:
No need to change this but FYI it's `helm` that uses `$HOME` when it pulls a chart.

Reply:
Updated to be more precise